resources/views/user/profile.blade.php

```diff
@@ -0,0 +1,95 @@
+@extends('user.layout')
+@section('content')
+<div class="container">
+    @include('status')
+    @include('message_validation')
+    <div class="row trans-form">
+        <div class="col-4 avatar">
+            <div class="card" style="width: 18rem;">
+                <img src="{{ asset('/images/'.$profile->image) }}" class="card-img-top" alt="Avatar">
+                <div class="card-body">
+                    <h5 class="card-title">{{ $profile->fullname }}</h5>
+                    <p class="card-text"></p>
+                    <form method="POST" action="{{ route('profiles.image') }}" enctype="multipart/form-data">
+                        @csrf
+                        <div class="custom-file">
+                            <input type="file" class="custom-file-input" id="customFile" name="image">
+                            <label class="custom-file-label" for="customFile">Choose file</label>
+                        </div>
+                        <button type="submit" class="btn btn-primary">Update Avatar</button>
+                    </form>                 
+                </div>
+            </div>
+        </div>
+        <form method="POST" action="{{ route('profiles.update', Auth::user()->id) }}" class="col-8 form-profile d-flex flex-column justify-content-around">
+            @csrf
+            @method('PUT')
+            <div class="form-group row">
+                <label for="fullname" class="col-sm-2 col-form-label">Full Name</label>
+                <div class="col-sm-8">
+                    <input type="text" class="form-control" name="fullname" id="fullname" value="{{ Auth::user()->name }}">
+                </div>
+            </div>
+
+            <div class="form-group row">
+                <label for="gender" class="col-sm-2 col-form-label">Gender</label>
+                <div class="col-sm-8">
+                    <select class="custom-select" id="gender" name="gender">
```

Comment:
in case validation fails or some errors occur, this will not keep what users has input, please fix